jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/internal/JRubyExecUtils.groovy

```diff
@@ -59,7 +59,7 @@ class JRubyExecUtils {
         String version = jrubyJarVersion(jar)
         if(!version) {return null}
 
-        Matcher matches = version =~ /(\d{1,2})\.(\d{1,3})\.(\d{1,3})/
+        Matcher matches = version =~ /(\d{1,2})\.(\d{1,3})\.(\d{1,3}).*/
```

Comment:
We can probably be slightly more clever about this, but I'll rasie a separate issue for that. I would rather see this merged.